docs/Tables.md

```diff
@@ -277,6 +278,61 @@ Result:
 All the possible layout values are described
 there: [`TableBordersLayout`](https://py-pdf.github.io/fpdf2/fpdf/enums.html#fpdf.enums.TableBordersLayout).
 
+## Set cell borders
+
+_New in [:octicons-tag-24: 2.8.2](https://github.com/py-pdf/fpdf2/blob/master/CHANGELOG.md)_
+
+```python
+pdf = FPDF()
+pdf.add_page()
+pdf.set_font("Times", size=16)
+with pdf.table() as table:
+    for data_row in TABLE_DATA:
+        row = table.row()
+        for datum in data_row:
+            row.cell(datum, border=1) # Left 
+pdf.output('table.pdf')
+```
+
+Result:
+
+![](table_with_cell_border_left.jpg)
+
+```python
+pdf = FPDF()
```

Comment:
```suggestion
from fpdf import FPDF

pdf = FPDF()
```
We try to make all code snippets autonomous: you can copy paste it in a `.py` script and it should just work!
That's why including the imports is important 🙂